src/containerapp/azext_containerapp/_params.py

```diff
@@ -338,7 +336,7 @@ def load_arguments(self, _):
 
     with self.argument_context('containerapp hostname list') as c:
         c.argument('name', id_part=None)
-        c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx))
```

Comment:
I want to confirm that deleting the default location from resource group will cause a breaking change to users?

Reply:
Yes, this is not a breaking change. The API accepts a `null` value for location, and the default location that was being chosen before was causing errors for users

Reply:
Correct - RG supports a lot more locations than ACA does @zhoxing-ms , hence this change. Thanks for calling this out & checking.